components/utility.css

```diff
@@ -0,0 +1,86 @@
+*{
+text-decoration: none;
+list-style: none;
+}
```

Comment:
Using * selector. These style will be applied to all selectors. Are you sure about this?

Reply:
yes !! actually i wanted to remove all text-decration and list-styling so tried using this !